backend/research/models.py

```diff
@@ -32,3 +32,50 @@ class ResearchScholar(Student):
     class Meta:
         verbose_name = "research scholar"
         default_related_name = "scholars"
+
+
+class Publication(models.Model):
+    CONFERENCE = "CNF"
+    JOURNAL = "JRN"
+    PUB_TYPE_CHOICES = [
+        (CONFERENCE, "conference"),
+        (JOURNAL, "journal"),
+    ]
+    COMMUNICATED = "COM"
+    REJECTED = "REJ"
+    ACCEPTED = "ACT"
+    STATUS_CHOICES = [
+        (COMMUNICATED, "communicated"),
+        (REJECTED, "rejected"),
+        (ACCEPTED, "accepted"),
+    ]
+    author = models.ManyToManyField(CustomUser)
+    title = models.CharField(max_length=100)
+    pub_type = models.CharField(max_length=11, choices=PUB_TYPE_CHOICES)
+    pub_date = models.DateField(null=True, blank=True)
+    status = models.CharField(max_length=20, choices=STATUS_CHOICES)
+    doi_number = models.CharField(max_length=255, null=True, blank=True, verbose_name="Digital Object Indentifier Number")
```

Comment:
too long? i don't think the numbers exceed 20-30 chars. We can keep it at 50 to be safe